samples/spm/README.rst

```diff
@@ -51,28 +56,13 @@ Requirements for the application firmware
 
 * The application firmware must be built as a non-secure firmware for the build target (for example, ``nrf9160dk_nrf9160_ns`` for the nRF9160 DK).
 
-Automatic building of SPM when SPM is enabled
-=============================================
-
-The sample is automatically built by the non-secure applications when the non-secure build target is used (for example, ``nrf9160dk_nrf9160_ns``).
-However, it is not a part of the non-secure application.
-
-Instead of programming SPM and the non-secure application at the same time, you might want to program them individually.
-To do this, disable the automatic building of SPM by setting the option ``CONFIG_SPM=n`` in the :file:`prj.conf` file of the application.
-
-If this results in a single-image build, the start address of the non-secure application will change.
-The security attribution configuration for the flash will change when SPM is not built as a sub-image.
-
```

Comment:
@hakonfam , @b-gent : As now TFM is the included as default for all applications and samples instead of SPM, shouldn't this section be removed?

Reply:
As this is part of the SPM documentation I guess the statement is still valid and should not be removed, since some users will still choose to use SPM instead of TF-M.